microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-106.md

```diff
@@ -0,0 +1,174 @@
+---
+title: What's new in DevTools (Microsoft Edge 106)
+description: "Introducing the Command Palette. Filter and automatically fix code issues in DevTools for Visual Studio Code. DevTools now uses less disk space when caching sourcemaps. Chromium color themes have been deprecated. Accessibility improvements in Focus Mode and high contrast mode. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/29/2022
+---
+# What's New in DevTools (Microsoft Edge 106)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Introducing the Command Palette
+
+<!-- Subtitle: Your favorite browser management and DevTools commands in one place. -->
+
+Command Palette makes it possible to access browser management and developer tools commands with just one keyboard shortcut: `Ctrl`+`Shift`+`Spacebar`.  To access the available browser management commands and DevTools commands, press `Ctrl`+`Shift`+`Spacebar`, and then start typing.  The commands are filtered based on your input characters:
```

Comment:
I'm concerned that readers will try this shortcut without first enabling the flag. Can we add something like:

```suggestion
In Microsoft Edge 106, you can now enable the Command Palette experiment. Command Palette makes it possible to access browser management and developer tools commands with just one keyboard shortcut: `Ctrl`+`Shift`+`Spacebar`.  To access the available browser management commands and DevTools commands, complete the following steps:
1. [Enable the Command Palette experiment](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette#enable-command-palette).
1. Press `Ctrl`+`Shift`+`Spacebar`
1. Start typing.  The commands are filtered based on your input characters:
```

Reply:
Revised to cover enabling.  
Separated the install step(s) from the using steps, and placed the flags screenshot first.  
Added inline steps to enable, since already have the flags page capture here - it's easy to avoid user having to leave What's New.  
Improved the UI description.